supysonic/jukebox.py

```diff
@@ -156,7 +156,7 @@ def __play_file(self):
 
         logger.debug("Start playing with command %s", args)
         try:
-            return Popen(args, stdin=DEVNULL, stdout=DEVNULL, stderr=DEVNULL)
-        except:
-            logger.exception("Failed running play command")
+            return run(args, check=True, stdout=DEVNULL, stderr=DEVNULL)
```

Comment:
There is no test on the jukebox, and this change from `Popen` to `run` will most likely break things.

`run` will wait for the spawned process to end before returning. This will prevent the jukebox to be able to stop playing/skip tracks during playback. It'll also raise errors as `run` returns a different type than `Popen` (a `CompletedProcess`) which lacks the required methods (`poll()`, `terminate()` and friends).

Reply:
Yep, I overlooked this for sure, sorry. Fixed now.